W-10705241-DOCS-2106-NullNothingInDW-duke (#88)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-core-types.adoc

| Null
| `type Null = Null`

> | A Null type, which is for `null` values.

Reviewer: ```suggestion
| A Null type, which represents the `null` value.
```
It represents just one value that is null (but that value can be used in multiple places). Just a detail.
